app/components/buttons/toggle-button.jsx

```
@@ -0,0 +1,38 @@
+export default function ToggleIconButton(props) {
+  const [active, setActive] = useState(false)
+
+  return (
+    <div className="toggle-icon-btn" role="button" onClick={() => {
+      setActive(!active)
+      if (props.onClick) {
+        props.onClick()
+      }
+    }}>
+      <div className={`state-layer relative h-8 w-8 md:h-12 md:w-12 rounded-full m-auto flex items-center justify-center ${props.className || ''}`}>
+        <md-ripple></md-ripple>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)